File: Projet/reactive-prog-frontend/src/models/comment.ts
import { Item } from "./item";
import { Customer } from "./customer";

export class Comment {

    idComment: number | undefined;
    comment: string;
    item: Item;
    customer: Customer;

    constructor() {
        this.comment = "";
        this.item = new Item();
        this.customer = new Customer();
    }

}

File: Projet/reactive-prog-backend/src/main/java/com/veille2021/reactiveprogbackend/model/Comment.java
package com.veille2021.reactiveprogbackend.model;

import lombok.Data;
import org.springframework.data.annotation.Id;

import javax.persistence.*;
import java.io.Serializable;

@Data
@Table(name = "comment")
public class Comment implements Serializable {

    @Id
    @Column
    private Integer idComment;

    @Column
    private String comment;

    @ManyToOne(fetch = FetchType.LAZY)
    //@Column(name = "id_customer")
    private Customer customer;

    @ManyToOne(fetch = FetchType.LAZY)
    //@Column(name = "id_item")
    private Item item;


}
